{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--3d37125a-bdb5-4397-9e40-1aab3fddbdf6",
    "created": "2024-08-13T16:07:27.973855Z",
    "modified": "2024-08-13T16:07:27.973859Z",
    "name": "No Title Available",
    "description": "In the DataBuf class in include/exiv2/types.hpp in Exiv2 0.26, an issue exists in the constructor with an initial buffer size. A large size value may lead to a SIGABRT during an attempt at memory allocation. NOTE: some third parties have been unable to reproduce the SIGABRT when using the 4-DataBuf-abort-1 PoC file.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2018-9145"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://bugzilla.novell.com/show_bug.cgi?id=1087879"
        },
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://github.com/xiaoqx/pocs/tree/master/exiv2"
        },
        {
            "name": "GLSA-201811-14",
            "tags": [
                "vendor-advisory",
                "x_refsource_GENTOO"
            ],
            "url": "https://security.gentoo.org/glsa/201811-14"
        },
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1564281"
        }
    ]
}